examples/default/main.tf

```diff
@@ -29,10 +29,11 @@ module "vpc" {
 }
 
 ##############################################################################
-# Update security group
+# Create security groups for demonstration purposes in this example
 ##############################################################################
 
-module "create_sgr_rule" {
+# Create a first security group with client security_group_rules + rules for internal ibm flows
```

Comment:
I think it would be better if we mention the 'optionally' here for the internal IBM rules. Create a first security group with the client security group rules and optionally the internal IBM flows rules.